downloadimagejs

```diff
@@ -0,0 +1,9 @@
+let imageForm = document.querySelector ('form');
+imageForm.addEventListener('submit', function(event){
+event.preventDefault();
+imageMaker.downloadImage();
+})
```

Comment:
I like these commands and organization you've made for the downloadImage command

Reply:
This looks good and ready for the master!

Reply:
I really like how this looks, the organization is key for the user.